README.md

```
@@ -97,7 +97,12 @@
 
 <p>
   <img align="right" src="https://img.shields.io/badge/Private-gray?style=flat" />
-  <picture><source media="(prefers-color-scheme: dark)" srcset=".github/icons/light/camera.svg"><img src=".github/icons/dark/camera.svg" width="14" height="14" style="vertical-align: middle;" /></picture><a href="https://github.com/surajmandalcell/instagram">instagram</a> - My Virtual Photography Site (Go+ReactESM) · <a href="https://instagram.mandalsuraj.com">instagram.mandalsuraj.com</a>
+  <picture><source media="(prefers-color-scheme: dark)" srcset=".github/icons/light/camera.svg"><img src=".github/icons/dark/camera.svg" width="14" height="14" style="vertical-align: middle;" /></picture><a href="https://github.com/surajmandalcell/instagram">instagram</a> - My Virtual Photography Site (Go+React/ESM) · <a href="https://instagram.mandalsuraj.com">instagram.mandalsuraj.com</a>
+</p>
+
+<p>
+  <img align="right" src="https://img.shields.io/badge/Private-gray?style=flat" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)